sktm/patchwork.py

```diff
@@ -480,14 +488,17 @@ def get_new_patchsets(self, db = None):
         #   the event_time from the last seen event can be used in the next
         #   query with a since parameter to only retrieve events that haven't
         #   been seen yet.
-        nsince = dateutil.parser.parse(self.since) + \
-                  datetime.timedelta(seconds=1)
+        nsince = dateutil.parser.parse(
+            self.since
+        ) + datetime.timedelta(seconds=1)
```

Comment:
Can we have this indented a couple more levels, so the closing parentheses is past the equals sign? Actually, can we have the rest of the expressions like that? Otherwise it's a bit difficult to distinguish this from a separate statement. Well, at least for me :)

Reply:
Shouldn't be hard to see the closing paren, but we can do the same as above if it's more readable for you:
```
nsince = (dateutil.parser.parse(self.since)
          + datetime.timedelta(seconds=1))
```